README.md

```diff
@@ -150,21 +150,21 @@ All operations are logged to:
 
 If you have test files (e.g., in `tests/`):
 ```bash
-uv run pytest --cov=src
+uv run pytest --cov=lib
 ```
 
 ## Troubleshooting
-- **ModuleNotFoundError:** Make sure you set `PYTHONPATH=src` when running the Shiny app.
+- **ModuleNotFoundError:** Use `uv run` to automatically handle Python path and dependencies.
 - **Port already in use:** Use a different port with `--port`.
 - **Data not found:** Ensure your `drug_data.csv` is in the correct `data/` directory.
 - **Dependency issues:** Reinstall requirements with `pip install -r requirements.txt` or `pip install -e .`
 - **Input file errors:** Check that your CSV file uses semicolon separators and has the correct column names or data format.
 
 ## Project Structure
 
-- `src/core/` — Core logic, calculation functions, and data utilities
-- `src/cli/` — Command-line interface entry point (`main.py`)
-- `src/shinyapp/` — Shiny for Python web app
+- `lib/src/` — Core logic, calculation functions, and data utilities
+- `app/cli/` — Command-line interface entry point (`main.py`)
+- `app/shiny/` — Shiny for Python web app
 - `data/` — Drug data CSV and reference files
```

Comment:
Let's use the name `db` in this refactor. 

A popular command line tool in the world of TBgenomics is called [tb-profiler](https://github.com/jodyphelan/TBProfiler) which follows these patterns and some similarity with that codebase will facilitate community contributions from people who're aware of that codebase.